_posts/2016-07-26-webhooks.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introducing webhooks
+author: Thomas Hipp
+layout: blogpost
+---
+
+Hello!
+
+I am pleased to announce one of the new features which will be shipped with
+Portus version 2.1.
+
+#### What are webhooks and how do they work?
+
+Simply put, a webhook is a user-defined HTTP callback which notifies a
+specified endpoint about some sort of change.
+In Portus, this change can be either a push or delete event, i.e. webhooks will
+be triggered when you push or delete an image.
+
+Creating a webhook in Portus is as simple as defining a URL.
+Although every HTTP request also requires a request method, this does not
+explicitly need to be set by the user, and defaults to `GET` in Portus.
+Another option, or better header, which has a default value is `content-type`.
+It defaults to `application/json` but can be changed if needed.
+Other customizable options include  username and password, and custom HTTP
+headers.
+Per default, these are left empty if not explicitly specified.
+Furthermore, each webhook can be temporary disabled if it's not needed, and be
+enabled again any time.
+Note: a newly created webhook is disabled and needs to be enabled before it can
+be used.
+
+As mentioned above, webhooks in Portus react to push and delete event.
+They are implemented at namespace level which means that the specified webhooks
+will be triggered for each repository in the namespace.
+Any kind of filtering therefore has to be done by the receiver.
+Should a specific webhook delivery fail or need to be retriggered, this can be
+done by via the web UI.
```

Comment:
Only: "Should a specific webhook delivery fail or need to be retriggered, this can be done by via the web UI.". For my non-english brain that's a bit difficult to read :)